<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="drawingArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editText_X">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>50</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editText_Y">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="parameter2EditText">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>500</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="parameter1EditText">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputYLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Point 1 Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputXLabel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>65</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Point 1 X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="parameter1Label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>59</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="parameter2Label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>460</y>
      <width>59</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mSetPointButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>150</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mDrawPolygonButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Polygon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mTranslationButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mTransformButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>600</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mScalingButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>320</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mRotationButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
